<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HokumCrypt</class>
 <widget class="QMainWindow" name="HokumCrypt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>89</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HokumCrypt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_browse">
        <property name="text">
         <string>Browse File</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="ui_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_crypt">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Crypt</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btn_decrypt">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Decrypt</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QProgressBar" name="progress_bar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QLabel" name="lbl_info">
        <property name="text">
         <string>Choose a file to encrypt or decrypt.</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
